src/overworld.c

```diff
@@ -835,7 +837,7 @@ static void mli0_load_map(u32 a1)
             LoadEventObjTemplatesFromHeader();
     }
 
-    v2 = is_map_type_1_2_3_5_or_6(gMapHeader.mapType);
+    v2 = IsMapTypeOutside(gMapHeader.mapType);
```

Comment:
Update this variable name to something more descriptive than `v2`, like `isOutdoors`